### PR TITLE
Bugfix/product dependency test plan

### PR DESCRIPTION
**Pull request by mayankchhajer**

### CODE REVIEW

**Comment by vikasbo** [comment]
Approved. 

@rinkykasyap Please add your approval once you have reviewed again.

Thanks.

**Comment by rinkykasyap** [comment]
@vikasbo Sir This PR is not tested completely for the Sanity and Functional Test Plan yet.

**Comment by vikasbo** [comment]
@mayankchhajer As per Rinky's comment here https://github.com/indianashu/WDAutomation/pull/39#issuecomment-453025963 , this PR needs to be tested for both Sanity and Functional Test plans. Please do so and confirm back the results.

File: src/test/java/productHistoricalTest/AddCustomerTest.java [hunk 1]
```diff
@@ -41,7 +43,8 @@ public void userSignin() throws Exception {
 			addCustomerPage.selectCurrency(rowNum);
 			addCustomerPage.navigateBottom();
 			addCustomerPage.clickSaveChangesButton();
-			addCustomerPage.verifyConfirmationMsg("Saved Customer " + (rowNum+1)  + "Successfully For Parent Company");
+			String name= driver.findElement(By.xpath("//tr[@class='active']//strong//span")).getText();
+			addCustomerPage.verifyConfirmationMsg("Saved " + name + " Successfully For Parent Company");
```
Comment (rinkykasyap) [inline src/test/java/productHistoricalTest/AddCustomerTest.java]: @mayankchhajer There is no need to pass the name in the message, So Please remove the String name variable.

File: src/test/java/productHistoricalTest/LoginSignupCompanyTest.java [hunk 2]
```diff
@@ -1,12 +1,7 @@
...
 import baseClassPackage.BaseClasses;
-import productHistorical.AddProduct1Page;
-import productHistorical.AddProduct4Page;
 import productHistorical.LoginSignupCompanyPage;
-
-import java.io.IOException;
-
 import org.openqa.selenium.WebDriver;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
```
Comment (rinkykasyap) [inline src/test/java/productHistoricalTest/LoginSignupCompanyTest.java]: @mayankchhajer Please do not pass any argument in the verifyConfirmationMsg()  message, Please revert your changes .

**Comment by vikasbo** [comment]
@rinkykasyap  Can you run this along with your code review to confirm that it runs well when there are no issues. Secondly, if you deliberately add an issue, it shows the error message in the report like we intend it to do?

In other words test it.

**Pull request merged**